src/wrapped/wrappedgdk3.c

```diff
@@ -181,22 +179,12 @@ EXPORT uint32_t my3_gdk_threads_add_timeout_full(x64emu_t* emu, int priotity, ui
     return my->gdk_threads_add_timeout_full(priotity, interval, findGSourceFunc(f), data, findGDestroyNotifyFct(d));
 }
 
-
 #define PRE_INIT    \
     if(box64_nogtk) \
         return -1;
 
-#define CUSTOM_INIT \
-    libname = lib->name;            \
```

Comment:
this line about libname disapeared?

Reply:
Yes, `libname` is unused in the entire file.